### PR TITLE
created leave model

### PR DESCRIPTION
**Pull request by Tadii3108**

### CODE REVIEW

File: backend/attendance/models.py [hunk 1]
```diff
@@ -119,3 +119,14 @@ def graph_dict(self):
             "comments": self.comments,
             "problems": self.reason_not_completed,
         }
+
+
+class Leave(models.Model, Mixins):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    LEAVE_TYPE = (("1", "Annual Leave"),("2", "Sick Leave"),("3", "Family Responsibility"))
```
Comment (Andy-Nkumane) [inline backend/attendance/models.py:126]: How about adding maternity and paternity leaves to the leave type

**Review by Andy-Nkumane — CHANGES_REQUESTED** [review]
I had a look at the current leave form and saw that it has `amount of days` - can you confirm if that field is required or not

**Comment by Tadii3108** [comment]
> I had a look at the current leave form and saw that it has `amount of days` - can you confirm if that field is required or not

I'm not sure I understand. Required as in if days as a whole are required or specifically the field `amount of days` is required *instead of* another possible way of having the days presented?

File: backend/attendance/models.py (hunk 1 above)
Comment (Tadii3108) [reply backend/attendance/models.py:126]: Wil do.

Comment (Tadii3108) [reply backend/attendance/models.py:126]: Wait, wouldn't those two - maternity and paternity leaves - fall under Family responsibility?

Comment (Andy-Nkumane) [reply backend/attendance/models.py:126]: Have a look at this https://docs.google.com/forms/d/e/1FAIpQLSdRy444hA3WR4Vul4nTXOuC0x9ZuA3TFBaB8aA-PPMDCeQN8g/viewform

**Comment by Andy-Nkumane** [comment]
> > I had a look at the current leave form and saw that it has `amount of days` - can you confirm if that field is required or not
> 
> I'm not sure I understand. Required as in if days as a whole are required or specifically the field `amount of days` is required _instead of_ another possible way of having the days presented?

The number of days for leave, excluding weekends and public holidays

**Review by Andy-Nkumane — CHANGES_REQUESTED** [review]
This is looking good, just a few things.

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**